docker/conf/docker-compose.demo.yml.dist

```
@@ -0,0 +1,61 @@
+version: "3.4"
+services:
+    postgres:
+        image: postgres:10.5-alpine
+        volumes:
+            - ./docker/postgres/postgres.conf:/var/lib/postgresql/data/postgresql.conf
+        environment:
+            - PGDATA=/var/lib/postgresql/data/pgdata
+            - POSTGRES_USER=root
+            - POSTGRES_PASSWORD=root
+            - POSTGRES_DB=shopsys
+
+    webserver:
+        image: nginx:1.13-alpine
+        depends_on:
+            - php-fpm
+        volumes:
+            - web-volume:/var/www/html/web
+            - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
+        ports:
+            - "8080"
+
+    php-fpm:
+        build:
+            context: .
+            dockerfile: docker/php-fpm/Dockerfile
+            target: production
+        volumes:
+            - web-volume:/var/www/html/web
```

```suggestion
            - ./web:/var/www/html/web
```
i think that this needs to be here this way

---

i liked the dedicated volumes more.